server-core/src/main/java/io/infectnet/server/core/ApplicationStarter.java

```diff
@@ -1,9 +1,20 @@
 package io.infectnet.server.core;
 
+import io.infectnet.server.controller.impl.RegistrationController;
+import io.infectnet.server.controller.RestController;
+import io.infectnet.server.controller.impl.TokenController;
+
 import javax.inject.Inject;
+import java.util.Arrays;
+import java.util.List;
 
 public class ApplicationStarter {
 
+    List<RestController> restControllers = Arrays.asList(
```

Comment:
This field should be private and should not be initialized inline. Please move the initialization to the constructor. Also add `final` modifier as this list should not be modified.